### ISSUE TITLE
Expose frame resolution from container.

### ISSUE THREAD

**Issue by dalecurtis** [AOMediaCodec/libavif#13]
If present it'd be nice to have the resolution information as known by the container so the image doesn't need to be decoded until necessary. On the web we are frequently in need of just the size.

**Comment by joedrago** [comment]
I was already parsing this data, I just wasn't surfacing it to the decoder. If we're decoding tracks instead of "items", there won't be such a value and they'll just be 0.

https://github.com/AOMediaCodec/libavif/commit/41700854b03341096687f7d97011d20c6fb0c803

**Comment by dalecurtis** [comment]
For tracks the information is in the tkhd box which does seem to be present in the avif test files. Can you also surface that information?

**Comment by dalecurtis** [comment]
Should also be in the stsd box, so whatever's easiest.

**Comment by joedrago** [comment]
Ah, you're right, my naming was a bit short-sighted there. I've renamed ispeWidth and ispeHeight to containerWidth and containerHeight, respectively, and I've documented them in the header as the "width and height as reported by the AVIF container". They should now offer the integer portion of the tkhd box's width/height when decoding tracks.

https://github.com/AOMediaCodec/libavif/commit/fc4144ee02112e97f360407cdb17a4359ada4a23
